src/main/java/org/wildfly/extension/elytron/ElytronOperationStepHandler.java

```diff
@@ -0,0 +1,27 @@
+package org.wildfly.extension.elytron;
```

Comment:
That's really weird. All the new source. Some default must changed in my IDE settings.